test_fractions.py

```diff
@@ -796,11 +797,50 @@ def test_pi_digits(self):
             self.assertEqual(ff.numerator, qf.numerator)
             self.assertEqual(ff.denominator, qf.denominator)
 
+class CImportTest(unittest.TestCase):
```

Comment:
This test file is almost entirely copied from the CPython test suite, so I would prefer not adding quicktions specific tests to it. Please create a new one, `src/test_quicktions.py`.